requirements/requirements_dev.txt

```diff
@@ -9,7 +9,6 @@ black
 pytest!=7.0.0
 pytest-cov
 pytest-django
-pytest-pythonpath
```

Comment:
This library requires an old version of pytest which is, of course, conflicting with other libraries here. I guess it just ignored the conflict up until this point. If you actually go to the project, it basically says it's old and don't use it anymore. This won't work with the conflict, and it seems we didn't need this either.